apiserver/facades/agent/uniter/uniter.go

```diff
@@ -428,34 +428,6 @@ func (u *UniterAPI) AvailabilityZone(ctx context.Context, args params.Entities)
 	return results, nil
 }
 
-// Resolved returns the current resolved setting for each given unit.
-func (u *UniterAPI) Resolved(ctx context.Context, args params.Entities) (params.ResolvedModeResults, error) {
```

Comment:
We should rev the facade to do this. But that would require spinning a new 3.6 release. Even though it's not used. Instead, can we keep the method and return NotSupported instead.

Reply:
Yep you're right.

Done